dpctl-capi/cmake/modules/FindLevelZero.cmake

```diff
@@ -25,8 +25,16 @@
 # LEVEL_ZERO_LIBRARY - the full path to the ze_loader library
 # TODO: Add a way to record the version of the level_zero library
 
+if(WIN32)
+    set L0_INCLUDE_DIR="%CD%\\level-zero\\include\\zet_api.h"
+endif()
```

Comment:
`L0_INCLUDE_DIR` should point to `%CD%`. And it should be set outside of cmake scripts. Better place is conda build scripts (bld.bat, build.sh).